example/js/BUILD.bazel

```diff
@@ -40,3 +30,12 @@ oci_tarball(
     image = ":image",
     repotags = ["example/js:latest"],
 )
+
+structure_test(
+    name = "test",
+    config = ["test.yaml"],
+    image = ":image",
+    target_compatible_with = [
+        "@platforms//os:linux",
```

Comment:
Why only Linux?

Reply:
container rule aka oci_image is not ready to handle transitions. will handle them #33